Cargo.toml

```diff
@@ -167,7 +167,7 @@ oxc_transform_napi = { version = "0.26.0" }
 
 [profile.release]
 codegen-units = 1
-debug         = false     # Set to `true` for debug information
+debug         = 1     # Set to `true` for debug information
```

Comment:
Do we need to keep this?

Reply:
No,  I am profiling with some screenshot, hang on for a few seconds.